tests/contrib/pymongo/test.py

```diff
@@ -406,6 +409,36 @@ def test_host_kwarg(self):
         assert client
 
 
+class TestPymongoPatchDisabledTracer(TestPymongoPatchDefault):
+    def get_tracer_and_client(self):
+        tracer = DummyTracer()
+        tracer.configure(enabled=False)
+        return tracer, self.get_client(tracer)
```

Comment:
You should be able to still use the original helper without the addition of `get_client()`.

Since we pass the `tracer` instance around as a reference (e.g. in the `Pin` usage) we can do:

```python
def get_tracer_and_client(self):
    tracer, client = super(TestPymongoPatchDisabledTracer, self).get_tracer_and_client()
    tracer.enabled = False  # we don't need `tracer.configure()` here, either works
    return tracer, client
```